product_docs/docs/edbcloud/beta/using_cluster/03_modifying_your_cluster/05_db_configuration_parameters.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: Modifying Database Configuration Parameters
+---
+
+The database parameters listed on the DB Configuration tab are also referred to as Grand Unified Configuration (GUC) variables. See [What Is a GUC Variable?](https://www.enterprisedb.com/blog/what-guc-variable) for more information.
+
+The list of parameters is populated based on the type of database you selected on the **Operational Settings** tab when you created your cluster. For more information about the parameters for your database type:
+
+- For Postgres parameters, see 
+   - [Setting Parameters](https://www.postgresql.org/docs/current/config-setting.html) and [Server Configuration](https://www.postgresql.org/docs/current/runtime-config.html) in the Postgres documentation.
+   - [postgresqlco.nf](https://postgresqlco.nf/) for more documentation and a tool to manage your configuration file.
+- For EDB Postgres Advanced Server, see [Summary of Configuration Parameters](https://www.enterprisedb.com/docs/epas/latest/epas_guide/03_database_administration/01_configuration_parameters/02_summary_of_configuration_parameters/#summary_of_configuration_parameters) and [Configuration Parameters](https://www.enterprisedb.com/docs/epas/latest/epas_guide/03_database_administration/01_configuration_parameters/).
+
+!!!note
+Not all database configuration parameters are supported by EDB Cloud. Some parameters are reserved for EDB to provide the managed database features of EDB Cloud such as `wal_level` and `restore_command`. 
```

Comment:
This reads like `wal_level` and `restore_command` are features of EDB Cloud, not GUCs 😁